templates/navigation-mobile-menu-items.twig

```diff
@@ -0,0 +1,63 @@
+{% for key,item in menu %}
+	{% set targetId = item.ID %}
+	{% if fn('get_post_meta', item.ID, '_menu_item_object_id', true ) == act_page_id %}
+		{% set link_ga_action = 'Act' %}
+	{% elseif fn('get_post_meta', item.ID, '_menu_item_object_id', true ) == explore_page_id %}
+		{% set link_ga_action = 'Explore' %}
+	{% else %}
+		{% set link_ga_action = item.title %}
+	{% endif %}
+	<li class="navigation-mobile-menu-item accordion-item {{ item.class }} {{ item == item.current ? 'active' : '' }}">
+		<div class="item-wrapper accordion-header" id="navigation-mobile-menu-accordion-{{ targetId }}">
+			<a
+				class="nav-link"
+				href="{{ item.get_link }}"
+				data-ga-category="Submenu Navigation"
+				data-ga-action="{{ link_ga_action }}"
+				data-ga-label="{{ page_category }}">
+					{{ item.title }}
+			</a>
+			{% if item.children is not empty %}
+				<button
+					class="collapsable-btn accordion-button collapsed"
+					type="button"
+					data-bs-toggle="collapse"
+					data-bs-target="#menu-{{ targetId }}"
+					aria-expanded="false"
+					aria-controls="menu-{{ targetId }}"
+				/>
+			{% endif %}
+		</div>
+
+		{% if item.children is not empty %}
+		<nav
+			id="menu-{{ targetId }}"
+			class="subitems-nav accordion-collapse collapse"
+			aria-labelledby="navigation-mobile-menu-accordion-{{ targetId }}"
+			data-bs-parent="#accordion-list"
+		>
+			<ul>
+			{% for key,item in item.children %}
+				<li class="accordion-body">
```

Comment:
Maybe you're missing the `active` class definition here (like on l.10) ? The menu doesn't mark the page I'm on in a submenu
![Screenshot from 2022-03-03 21-40-28](https://user-images.githubusercontent.com/617346/156649237-2759f65f-f0de-4bc6-a7b6-f6f9de5cee62.png)

I don't know what are the design requirements, but you could also mark the parent item as `active` same as the current sub-item using the (bool) `current_item_parent` property; so something like that for example
```diff
diff --git a/templates/navigation-mobile-menu-items.twig b/templates/navigation-mobile-menu-items.twig
index 2cc41d36..61362946 100644
--- a/templates/navigation-mobile-menu-items.twig
+++ b/templates/navigation-mobile-menu-items.twig
@@ -7,7 +7,7 @@
 	{% else %}
 		{% set link_ga_action = item.title %}
 	{% endif %}
-	<li class="navigation-mobile-menu-item accordion-item {{ item.class }} {{ item == item.current ? 'active' : '' }}">
+	<li class="navigation-mobile-menu-item accordion-item {{ item.class }} {{ item.current or item.current_item_parent ? 'active' : '' }}">
 		<div class="item-wrapper accordion-header" id="navigation-mobile-menu-accordion-{{ targetId }}">
 			<a
 				class="nav-link"
@@ -38,7 +38,7 @@
 		>
 			<ul>
 			{% for key,item in item.children %}
-				<li class="accordion-body">
+				<li class="accordion-body  {{ item.class }} {{ item.current ? 'active' : '' }}">
 					{% if fn('get_post_meta', item.ID, '_menu_item_object_id', true ) == act_page_id %}
 						{% set link_ga_action = 'Act' %}
 					{% elseif fn('get_post_meta', item.ID, '_menu_item_object_id', true ) == explore_page_id %}
```

The other issue being that the `active` class of an item seems to apply its style to sub-items, so we should also restrict it I guess
![Screenshot from 2022-03-03 21-03-21](https://user-images.githubusercontent.com/617346/156650282-ceeb8913-494c-42b3-9345-d8cb5d589a82.png)